Formula/fabric-installer.rb

```diff
@@ -0,0 +1,21 @@
+class FabricInstaller < Formula
+  desc "Installer for Fabric for the vanilla launcher"
+  homepage "https://fabricmc.net/"
+  url "https://maven.fabricmc.net/net/fabricmc/fabric-installer/0.7.3/fabric-installer-0.7.3.jar"
+  sha256 "a3902483785e6d80e794c81ba372c91d244684827cce176c1a9d605aec416fa7"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    libexec.install "fabric-installer-#{version}.jar"
+    bin.write_jar_script libexec/"fabric-installer-#{version}.jar", "fabric-installer"
+  end
+
+  test do
+    system "#{bin}/fabric-installer", "server"
+    system "test", "-f", "fabric-server-launch.jar"
```

Comment:
```suggestion
    assert File.exists?("fabric-server-launch.jar")
```

Reply:
I think that method is deprecated and replaced by `File.exist`, but I think that `File.file` would be more accurate to `test -f` in this case. Correct me if I'm wrong though.

```suggestion
    assert File.file?("fabric-server-launch.jar")
```

Reply:
I'm not sure what that one returns. In the end we mostly want to check that it exists

Reply:
I believe `File.exist` can include directories. I suppose it would work if you're trying to avoid testing the formula itself though.

Reply:
I will settle with `File.exist`.

```suggestion
    assert File.exist?("fabric-server-launch.jar")
```

Reply:
Fixed in 19ae56466521b33075ea325e47ce64cc869bec15